### ISSUE TITLE
Blood Magic Integration

### ISSUE THREAD

**Issue by Banjooie** [Chisel-Team/Chisel#298]
Pursuant to a conversation on IRC: It would be really neat if I could use Chisel netherrack and dirt to improve the Blood Magic Tranquility Altar. Apparently it requires these shenanigans:

My dim understanding is you do something with this thing here
https://github.com/WayofTime/BloodMagic/blob/1.9/src/main/java/WayofTime/bloodmagic/api/incense/ITranquilityHandler.java

Using relevant values from this thing here
https://github.com/WayofTime/BloodMagic/blob/1.9/src/main/java/WayofTime/bloodmagic/incense/TranquilityHandlers.java

And then registering it here.
https://github.com/WayofTime/BloodMagic/blob/1.9/src/main/java/WayofTime/bloodmagic/api/incense/IncenseTranquilityRegistry.java#L15

Basically I want to stab myself more efficiently while making things super pretty.

**Comment by Drullkus** [comment]
> Basically I want to stab myself more efficiently while making things super pretty.

10/10

**Comment by Drullkus** [comment]
I think it's done? Idk how to test.

**Comment by tterrag1098** [comment]
Released
